### ISSUE TITLE
Component: Panel Menu Accessibility Issue: Interactive Controls Must Not Be Nested

### ISSUE THREAD

**Issue by WTW-Jacob-Evans** [primefaces/primeng#15085]
### Describe the bug

Using Axe DevTools as part of accessibility work, the PanelMenu component raises an issue relating to nested controls.

Is it possible for this to be fixed?

![image](https://github.com/primefaces/primeng/assets/163871134/ccf11b69-f6da-4d32-8021-276f5e73d33d)

![image](https://github.com/primefaces/primeng/assets/163871134/2cdded10-81c7-43c5-8d52-a24bf053a715)

### Environment

Chrome,
Windows 10

### Angular version

17.2.3

### PrimeNG version

17.10.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.18.0

**Comment by cetincakiroglu** [comment]
Hi,

We've tried to replicate with Axe Devtools, however mentioned error didn't show up. Apart from unrelated issues from the page, we couldn't catch the mentioned issue. Could you please provide a stackblitz reproducer so we can identify the issue?

![Screenshot 2024-03-26 at 15 31 44](https://github.com/primefaces/primeng/assets/69278826/456cc211-36df-4fa6-994d-6b87be280967)

**Comment by WTW-Jacob-Evans** [comment]
Hi, sorry for the late reply; whilst I am unable to replicate it, I can see the element raising the issue is here 
![image](https://github.com/primefaces/primeng/assets/163871134/8b30a87e-ad81-4288-9403-401ff8698fa6)
Which is the button inside the panelMenu component. Do you have any suggestions why it is raising it as an issue here but not elsewhere?

**Comment by WTW-Steve-Martin** [comment]
Hi,
Please can I have an update on this bug please.
I check `17.18.1` and this still seems to be an issue.

**Comment by mehmetcetin01140** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you open a new issue so we can include it in our roadmap?

Thanks a lot for your understanding!
Best Regards,
